### ISSUE TITLE
Training collapse for small batch

### ISSUE THREAD

**Issue by YuhsiHu** [cvlab-kaist/C3G#3]
Hi! Thanks for releasing C3G and the code.

I’m trying to reproduce training, but I’m facing a consistent divergence when training on a **single RTX 4090 (24GB)**. The paper mentions training on **8×H100** with batch_size=8 per GPU. I only have 1×4090, so I set batch_size=1 and scaled down the learning rate accordingly. However, training collapses after some iterations: the predicted depth becomes mostly/all negative, and therefore the rendered images turn completely black.

### My setup

* GPU: 1× RTX 4090 (24GB)
* Batch size: 1
* Dataset: re10k
* Command:
```bash
python -m src.main +training=gaussian_head
```
* Config changes compared to the default:

  * train batch_size: 4 -> 1
  * optimizer learning rate: 1e-4 -> 1.6e-5

### What exactly goes wrong

After ~2k steps:

gaussians.means, the ratio of z<=0 becomes 100%

### Questions

1. Is there any suggested way to adjust training for single-GPU / small batch?
2. Does the method rely critically on the multi-GPU for stability?
3. Are there known stability tricks to prevent negative Z / black rendering collapse?

Thanks a lot for any guidance!

**Comment by XuanyuZhang** [comment]
Meet the same question
